config/jobs/kubernetes/sig-cloud-provider/aws/kops/kops-periodics-old.yaml

```diff
@@ -1,181 +1,246 @@
+      preset-service-account: "true"
+      preset-aws-ssh: "true"
+      preset-aws-credential: "true"
+    decorate: true
+    decoration_config:
+      timeout: 140m
+    spec:
+      containers:
+        - command:
+            - runner.sh
+            - /workspace/scenarios/kubernetes_e2e.py
+          args:
+            - --timeout=140
+            - --bare
+            - --scenario=kubernetes_e2e
+            - --
+            - --cluster=e2e-kops-aws-k8s-1.13.test-cncf-aws.k8s.io
+            - --deployment=kops
+            - --env=KUBE_SSH_USER=admin
+            - --extract=ci/latest
+            - --ginkgo-parallel
+            - --kops-args=---kubernetes-version=1.13
+            - --kops-version=https://storage.googleapis.com/kops-ci/bin/latest-ci.txt
+            - --provider=aws
+            - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|Services.*functioning.*NodePort
+            - --timeout=120m
+          image: gcr.io/k8s-testimages/kubekins-e2e:latest-1.13
+          imagePullPolicy: Always
+    annotations:
+      testgrid-dashboards: google-aws, sig-cluster-lifecycle-kops
+      testgrid-tab-name: kops-aws-k8s-1.13
+
+  - interval: 4h
+    name: e2e-kops-aws-k8s-1-14
+    labels:
+      preset-service-account: "true"
+      preset-aws-ssh: "true"
+      preset-aws-credential: "true"
+    decorate: true
+    decoration_config:
+      timeout: 140m
+    spec:
+      containers:
+        - command:
+            - runner.sh
+            - /workspace/scenarios/kubernetes_e2e.py
+          args:
+            - --timeout=140
+            - --bare
+            - --scenario=kubernetes_e2e
+            - --
+            - --cluster=e2e-kops-aws-k8s-1.14.test-cncf-aws.k8s.io
+            - --deployment=kops
+            - --env=KUBE_SSH_USER=admin
+            - --extract=ci/latest
+            - --ginkgo-parallel
+            - --kops-args=---kubernetes-version=1.14
+            - --kops-version=https://storage.googleapis.com/kops-ci/bin/latest-ci.txt
+            - --provider=aws
+            - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|Services.*functioning.*NodePort
+            - --timeout=120m
+          image: gcr.io/k8s-testimages/kubekins-e2e:latest-1.14
+          imagePullPolicy: Always
+    annotations:
+      testgrid-dashboards: google-aws, sig-cluster-lifecycle-kops
+      testgrid-tab-name: kops-aws-k8s-1.14
+
+  - interval: 4h
+    name: e2e-kops-aws-k8s-1-15
+    labels:
+      preset-service-account: "true"
+      preset-aws-ssh: "true"
+      preset-aws-credential: "true"
+    decorate: true
+    decoration_config:
+      timeout: 140m
+    spec:
+      containers:
+        - command:
+            - runner.sh
+            - /workspace/scenarios/kubernetes_e2e.py
+          args:
+            - --timeout=140
+            - --bare
+            - --scenario=kubernetes_e2e
+            - --
```

Comment:
for pod utilities you'll need to remove all the args before (and including) the double hyphen.

also a nit: in the other kops job config files, lists aren't indented an extra two spaces, so they're like this:

```yaml
containers:
- command:
```

rather than this:

```yaml
containers:
  - command:
```

It might be worth being consistent if your editor supports it easily. not a big deal though.